dvc/repo/metrics/show.py

```diff
@@ -152,15 +152,16 @@ def _read_metric(fd, typ=None, xpath=None, fname=None, branch=None):
         return None
 
 
-def _collect_metrics(repo, path, recursive, typ, xpath, branch):
+def _collect_metrics(repo, targets, recursive, typ, xpath, branch):
     """Gather all the metric outputs.
 
     Args:
-        path (str): Path to a metric file or a directory.
+        targets (list): List of paths to metric files or directories.
+            If there's no target, collect all metrics.
         recursive (bool): If path is a directory, do a recursive search for
             metrics on the given path.
-        typ (str): The type of metric to search for, could be one of the
-            following (raw|json|tsv|htsv|csv|hcsv).
+        typ (str): The type that will be used to interpret the metric file,
+            one of the followings - (raw|json|tsv|htsv|csv|hcsv).
         xpath (str): Path to search for.
         branch (str): Branch to look up for metrics.
```

Comment:
Btw, deepsource reports that `branch` is not used here. Let's remove it while we are at it.